clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -1750,11 +1982,11 @@ Error linkBitcodeFiles(SmallVectorImpl<OffloadFile> &InputFiles,
 
   // We assume visibility of the whole program if every input file was bitcode.
   auto Features = getTargetFeatures(BitcodeInputFiles);
-  auto LTOBackend = Args.hasArg(OPT_embed_bitcode) ||
-                            Args.hasArg(OPT_builtin_bitcode_EQ) ||
-                            Args.hasArg(OPT_clang_backend)
-                        ? createLTO(Args, Features, OutputBitcode)
-                        : createLTO(Args, Features);
+  auto LTOBackend =
```

Comment:
This does not look like a SYCL specific change (atleast on face value). Should this go upstream?

Reply:
It is not technically SYCL specific, but in this PR SYCL is the only client using the new arguments.